app/views/tag/_contribute.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="row">
+  <div class="col-md-9" style="padding-left: 46px">
+
+    <% if @tag %>
+
+        <% if @users %>
+            <div class="row" id="highlight" style=" text-align: center;">
+              <h3 class="col-md-4 col-xs-12 col-sm-4"><%= @users.length || 0 %> contributors</h3>
+              <h3 class="col-md-4 col-xs-12 col-sm-4"><%= @notes2.length %> research notes</h3>
+              <h3 class="col-md-4 col-xs-12 col-sm-4"><%= @wikis.length %> wiki pages</h3>
+            </div>
+        <% end %>
+
+    <% end %>
+
+    <% if @notes2.nil? || @notes2.length == 0 %>
```

Comment:
This looks great, thank you! 

I'm wondering if we could pull out this section or even more, which is duplicated with this:

https://github.com/publiclab/plots2/blob/ceb7d05404b4619c7fa8cd8cae5e8215798cb02c/app/views/tag/contributors.html.erb#L71-L89

And use a partial to provide both? Like, `/app/views/tag/_contributors.html.erb` -- using `<%= render partial: "tag/contributors" %>`? What do you think?

Reply:
Good idea !!